mlrun/serving/v2_serving.py

```diff
@@ -299,6 +312,35 @@ def __init__(self, model, context, output_stream=None):
         self._batch_iter = 0
         self._batch = []
 
+    def init_endpoint_record(self):
+        project, uri, tag, hash_key = parse_versioned_object_uri(self.function_uri)
```

Comment:
what about exception handling, e.g. v3io doesnt exist (in open source) or IO error..